src/index.js

```diff
@@ -3,20 +3,33 @@ import ReactDOM from "react-dom";
 import GlobalStyles from "./styles";
 import Pages from "./pages";
 import reportWebVitals from "./reportWebVitals";
+import {
+  ApolloClient,
+  InMemoryCache,
+  ApolloProvider,
+  gql,
+} from "@apollo/client";
 
-// const client = new ApolloClient({
-//   uri: "http://localhost:4000",
-//   cache: new InMemoryCache(),
-// });
-// const root = ReactDOM.createRoot(document.getElementById('root'));
-// root.render(
-ReactDOM.render(
-  <React.StrictMode>
+const client = new ApolloClient({
+  uri: "http://localhost:4000",
```

Comment:
@DanielStamper I would suggest to create a `.env` in the project root with var:

```
REACT_APP_GRAPHQL_API_URL=http://localhost:4000/graphql
```

Then you can use it here like


```suggestion
  uri: process.env.REACT_APP_GRAPHQL_API_URL,
```